src/App.js

```diff
@@ -1,29 +0,0 @@
-import logo from './logo.svg';
```

Comment:
There are two important changes we need in the App.js file 🤓 

- The `App.js`(main component) should have the imported `Calculator.js` component rather than the `index.js`🚀
- The `App.js` component should be class-based so that we meet the project requirements ✔

Kindly address this issues 🔥